<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cusconfirmdialog</class>
 <widget class="QDialog" name="cusconfirmdialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="confirmBtn">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectBtn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>87</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>reject</string>
   </property>
  </widget>
  <widget class="QLabel" name="vidLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>vid</string>
   </property>
  </widget>
  <widget class="QLabel" name="cidLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>cid</string>
   </property>
  </widget>
  <widget class="QLabel" name="wtimeLabel">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>63</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>wtime</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
